### ISSUE TITLE
Feature: Remove whitespace

### ISSUE THREAD

**Issue by MicahElliott** [Lindydancer/nocomments-mode#2]
I find the whitespace left intact distracting. Would be great if it could all be hidden.

**Comment by monkeyjunglejuice** [comment]
Yeah, my first thought. 😆 That would be amazing. Second idea - handle comments and docstrings separately. Might be more involved to handle docstrings though, depending how it's implemented. I'm not aware of any "folding" packages that let you target comments and docstrings out of the box.
